### PR TITLE
Deprecate the stdweb backend

### PR DESCRIPTION
**Pull request by ryanisaacg**
- [ ] Tested on all platforms changed
- [ ] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] `cargo doc` builds successfully
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [ ] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

File: src/platform_impl/web/mod.rs [hunk 1]
```diff
@@ -29,6 +29,10 @@ mod backend;
 
 #[cfg(feature = "stdweb")]
 #[path = "stdweb/mod.rs"]
+#[cfg_attr(
+    feature = "stdweb",
+    deprecated(since = "0.23.0", note = "Please migrate to web-sys over stdweb")
+)]
```
Comment (chrisduerr) [inline src/platform_impl/web/mod.rs:35]: Wouldn't it be easier to put this at the top of the backend module?

Comment (ryanisaacg) [reply src/platform_impl/web/mod.rs:35]: 🤷, I'm not sure it changes much either way.

Comment (chrisduerr) [reply src/platform_impl/web/mod.rs:35]: You wouldn't need the feature check? Just seems cleaner to me.

Comment (ryanisaacg) [reply src/platform_impl/web/mod.rs]: Good point, thanks 👍

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**